packages/ui/vitest.config.ts

```diff
@@ -1,6 +1,8 @@
-import { defineConfig } from 'vitest/config';
+import { defineConfig, UserConfig } from 'vitest/config';
+import svgr from 'vite-plugin-svgr';
 
 export default defineConfig({
+  plugins: [svgr({ exportAsDefault: true })] as UserConfig['plugins'],
```

Comment:
I guess this typecast isn't needed anymore as removing it has no negative effects. Maybe we can remove it from other places as well.

```suggestion
  plugins: [svgr({ exportAsDefault: true })],
```